symbology/dictionary-symbol-sample/src/main/java/com/esri/samples/symbology/SymbolDictionarySample.java

```diff
@@ -0,0 +1,228 @@
+      searchFields[row] = new TextField();
+      searchPane.add(nameLabel, 0, row);
+      searchPane.add(searchFields[row], 1, row++);
+    }
+
+    // searches for any dictionary symbols that match text from text fields
+    Button searchButton = new Button("Search for Symbols");
+    searchButton.setMaxWidth(Double.MAX_VALUE);
+    searchButton.setOnAction(this::handleSearchAction);
+    searchPane.add(searchButton, 1, row);
+
+    // clears all of the search results
+    Button clearButton = new Button("Clear");
+    clearButton.setOnAction(this::handleClearAction);
+    searchPane.add(clearButton, 2, row++);
+
+    // displays number of search results
+    Label resultsLabel = new Label("Result(s) Found: ");
+    resultsFoundText = new Text();
+    searchPane.add(resultsLabel, 0, row);
+    searchPane.add(resultsFoundText, 1, row);
+
+    return searchPane;
+  }
+
+  /**
+   * Control that displays all search results in a scroll area.
+   * 
+   * @return control for displaying symbol dictionary search results
+   */
+  private Control createResultsControl() {
+    resultsPane = new ScrollPane();
+    resultsPane.setHbarPolicy(ScrollPane.ScrollBarPolicy.NEVER);
+    VBox.setVgrow(resultsPane, Priority.ALWAYS);
+
+    // displays search results
+    searchResultsBox = new VBox(10);
+
+    return resultsPane;
+  }
+
+  /**
+   * Searches through the symbol dictionary using the text from the search fields.
+   * <p>
+   * An image of the symbol(s) found is shown, along with its full name, tags, symbol class, category, and key.
+   * 
+   * @param event action from search button
+   */
+  private void handleSearchAction(ActionEvent event) {
+    searchParameters.getNames().add(searchFields[0].getText());
+    searchParameters.getTags().add(searchFields[1].getText());
+    searchParameters.getSymbolClasses().add(searchFields[2].getText());
+    searchParameters.getCategories().add(searchFields[3].getText());
+    searchParameters.getKeys().add(searchFields[4].getText());
+
+    try {
+      // searching for any matches
+      ListenableFuture<List<StyleSymbolSearchResult>> searchResult =
+          dictionarySymbol.searchSymbolsAsync(searchParameters);
+      List<StyleSymbolSearchResult> symbolResults = searchResult.get();
+      resultsFoundText.setText("" + symbolResults.size());
+
+      // reset result so duplicates are not added
+      searchResultsBox = new VBox(10);
+      // display each result found
+      symbolResults.forEach(symbolResult -> {
+        Text description = new Text("Name: " + symbolResult.getName()
+            + "\nTags: " + symbolResult.getTags()
+            + "\nSymbol Classe: " + symbolResult.getSymbolClass()
+            + "\nCategory: " + symbolResult.getCategory()
+            + "\nKey: " + symbolResult.getKey());
+
+        ImageView symbolImage = new ImageView();
+        symbolImage.setFitWidth(40);
+        // Text symbol are not Cim Symbol
+        if (!symbolResult.getCategory().startsWith("Text")) {

```

Comment:
(not for sure if you saw this comment the first time)
@vGandhi 
Heard that you know a little something about Dictionary Symbol. I have created this sample that searched and display symbols that it finds, however I can't get it to display any symbols who's category starts with text. Was seeing if there is a different way to display those or if I need to make an issue for that.